### PR TITLE
pos_payment_terminal + pos_customer_display: simplify JS code

### PR DESCRIPTION
**Pull request by alexis-via**
Move description from **openerp**.py to README.rst
Update demo data

### CODE REVIEW

**Comment by alexis-via** [comment]
I still haven't tested this new code with real hardware... if someone has access to real hardware and can test this new version, your help is welcomed.

File: pos_payment_terminal/pos_payment_terminal_demo.xml [hunk 1]
```diff
@@ -6,5 +6,9 @@
     <field name="payment_mode">check</field>
 </record>
 
+<record id="point_of_sale.pos_config_main" model="pos.config">
+    <field name="iface_payment_terminal" eval="True"/>
+</record>
```
Comment (legalsylvain) [inline pos_payment_terminal/pos_payment_terminal_demo.xml:11]: Thanks !

**Comment by legalsylvain** [comment]
Hi, 
As you're refactoring the code, could you do a few "git mv" to follow folder conventions ?
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#directories

Unfortunately, I don't have the hardware, but I'll buy one in a while.

**Comment by sebastienbeau** [comment]
I can test it tomorow if you want

**Comment by sylvainc** [comment]
LGTM :+1:

**Comment by alexis-via** [comment]
Tested with real hardware this morning: everything works well :+1:

**Comment by bealdav** [comment]
:+1:

**Pull request merged**